### PR TITLE
[WIP] update edit saved payment page to allow for PayPal instruments

### PR DESCRIPTION
**Pull request by leeBigCommerce**
#### What?

To allow users to edit their newly save-able PayPal instruments

#### Tickets / Documentation

- https://jira.bigcommerce.com/browse/PAYMENTS-4975

### CODE REVIEW

**Comment by leeBigCommerce** [comment]
Dependant on work to fulfil the new "account_payment_methods_v2" feature

**Comment by Tharaae** [comment]
@leeBigCommerce Isn't this PR is now already included the previous one?
https://github.com/bigcommerce/cornerstone/pull/1603

File: templates/pages/account/edit-payment-method.html [hunk 1]
```diff
@@ -12,43 +12,61 @@ <h2 class="page-heading">{{lang 'forms.payment_methods.edit.heading'}}</h2>
...
-                <dl class="paymentMethodForm-details">
-                    <dt class="paymentMethodForm-details-term">{{lang 'account.payment_methods.expiration'}}</dt>
-                    <dd class="paymentMethodForm-details-description">{{customer.payment_methods.selected_payment_method.expiry_month}}/{{customer.payment_methods.selected_payment_method.expiry_year}}</dd>
-                </dl>
-                <div class="form-field">
-                    <input type="checkbox" value="default_instrument" name="default_instrument" id="default_instrument" data-label="{{lang 'forms.payment_methods.default_instrument'}}" class="form-checkbox" {{#if customer.payment_methods.selected_payment_method.default_instrument}}checked{{/if}}>
-                    <label class="form-label" for="default_instrument">{{lang 'forms.payment_methods.default_instrument'}}</label>
-                </div>
-            </div>
+        {{#with customer.payment_methods.selected_payment_method}}
```
Comment (bc-marquis-ong) [inline templates/pages/account/edit-payment-method.html]: Yeah, this is not right. Should https://github.com/bigcommerce/cornerstone/pull/1603 takes precedence?

**Comment by leeBigCommerce** [comment]
Superseded by https://github.com/bigcommerce/cornerstone/pull/1603

**Pull request closed without merging**